stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -1801,13 +1884,13 @@ function log(A0::UpperTriangular{T}) where T<:BlasFloat
          2.060962623452836e-001,
          2.879093714241194e-001]
     tmax = size(theta, 1)
-    n = size(A0, 1)
-    A = copy(A0)
+    n = size(A, 1)
     p = 0
     m = 0
 
-    # Compute repeated roots
-    d = complex(diag(A))
+    # Find s0, the smallest s such that the ρ(triu(A)^(1/2^s) - I) ≤ theta[tmax], where ρ(X)
```

Comment:
This maybe should be `ρ(A^(1/2^s)`, but the algorithm in the paper says to use the approach in Algorithm 4.1, which is for upper triangular `A`.